### PR TITLE
Epic listing and instance support added

### PR DESCRIPTION
**Pull request by morlandi**
Thank you for your great job.
Please give me advices on how to complete the PR for inclusion in main project, if useful

### CODE REVIEW

**Comment by yakky** [comment]
@morlandi nice! I will happily review this and will provide help and hints in the next few days
I am wondering if this can be integrated with #51

**Comment by morlandi** [comment]
@yakky I did notice the open PR #51 before starting my customization, and had a quick look at it.
However, as no support was available for listing, I decided to rather start by cloning a model already existing in the main branch.
By the way, I'm currently using my fork in a small script used to extract a printable HTML document from a Taiga project: https://github.com/morlandi/taiga-print-report
Eventually, I would rather refer to the mail python-taiga project for better sharing.

**Comment by morlandi** [comment]
@yakky or @fmarco ... any suggestion on I could obtain a sort of UserStory.list_comments() functionality ? Should I lookup in Issues, IssueAttributes, History or what ?
Thanks in advance for attention

**Comment by yakky** [comment]
I am currently AFK, I'll post here some hints as soon as I can

**Comment by morlandi** [comment]
No hurry, take your time

**Comment by morlandi** [comment]
knock knock 😀

**Comment by morlandi** [comment]
Thank you @fmarco , sooner or later I will try this 😀

**Comment by yakky** [comment]
@marco-silva0000 nice! Please check with @morlandi which is also at the djangocon eu sprints venue

**Comment by morlandi** [comment]
@marco-silva0000 time is running out, must catch my flight back to Italy ... let's' keep in touch ;)

**Comment by yakky** [comment]
@rafleze @morlandi is there a way we can merge #77 and #66 they both needs the improvements of the other
Let's talk about the way to cooperate to get both merged

**Comment by morlandi** [comment]
@yakky and @rafleze, I manually merged PR #77 into #66 

Also, I adjusted test_epics.py contributed by @msilva ... this is something I'm not totally comfortable with ... just made sure all test are running.

Code coverage is still suffering, I'm afraid.

@yakky could you review the updated PR ?

File: tests/test_epics.py [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+    # @patch('taiga.requestmaker.RequestMaker.post')
+    # def test_import_epic(self, mock_requestmaker_post):
+    #     rm = RequestMaker('/api/v1', 'fakehost', 'faketoken')
+    #     Epics(rm).import_(1, 'Epic 1', 'New')
+    #     mock_requestmaker_post.assert_called_with(
+    #         '/{endpoint}/{id}/{type}', payload={'status': 'New', 'project': 1,
+    #                                             'subject': 'Epic 1'},
+    #         endpoint='importer', type='ep', id=1
+    #     )
```
Comment (yakky) [inline tests/test_epics.py]: @morlandi could you remove the commented code?

File: taiga/models/models.py [hunk 2]
```diff
@@ -267,6 +267,140 @@ class UserStoryAttachments(Attachments):
...
+    #         {
+    #             'project': project,
+    #             'subject': subject,
+    #             'status': status
+    #         }
+    #     )
+    #     response = self.requester.post('/{endpoint}/{id}/{type}',
+    #                                    endpoint="importer", id=project,
+    #                                    type="ep", payload=attrs)
+    #     return self.instance.parse(self.requester, response.json())
```
Comment (yakky) [inline taiga/models/models.py]: @morlandi could you remove the commented code?

**Comment by yakky** [comment]
Fix #77

**Comment by morlandi** [comment]
@yakky Comments removed. PR updated
ciaoooo

**Pull request merged**